pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/util/JsonResponseTransformer.java

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.dsm.route.util;
+
+import org.broadinstitute.dsm.util.proxy.jackson.ObjectMapperSingleton;
+import spark.ResponseTransformer;
+
+public class JsonResponseTransformer implements ResponseTransformer {
```

Comment:
We have `JsonTransformer` and now `JsonResponseTransformer`, both of which implement `ResponseTransformer`.  How about if we change `JsonTransfomer` to `GsonResponseTransformer` and/or this one to `JacksonResponseTransformer`?

Reply:
How about updating this class to reflect its jackson-ness, but leaving the gson transformer class as-is?

Reply:
Erf. I already made the change.
Happy to revert if that makes more sense.